dlrover/python/master/node/job_manager.py

```diff
@@ -194,21 +194,21 @@ def _init_training_node_manager(self):
         self._chief_manager = ChiefManager(
             self._job_nodes.get(NodeType.CHIEF, {}),
             self._job_resource,
-            self._ps_relaunch_max_num,
```

Comment:
Why not keep _ps_relaunch_max_num？

Reply:
There are `_ps_relaunch_max_num` for the PS and `_relaunch_on_worker_failure` for the worker. It is a bug to set `_ps_relaunch_max_num`  to workers.

Reply:
> Why not keep _ps_relaunch_max_num？

There are `_ps_relaunch_max_num` for the PS and `_relaunch_on_worker_failure` for the worker. It is a bug to set `_ps_relaunch_max_num`  to workers.